1.9/overview/architecture/distributed-process-management.md

```diff
@@ -0,0 +1,83 @@
+---
+post_title: Distributed Process Management
+nav_title: Distributed Process Management
+menu_order: 5
+---
+
+This section describes the management of processes in a DC/OS cluster, from the resource allocation to the execution of a process.
+
+At a high level, this interaction takes place between the DC/OS components when a user launches a process. Communication occurs between the different layers, such as the user interacting with the scheduler, and within a layer, for example, a master communicating with agents.
+
+![Concept of distributed process management in DC/OS](../img/dcos-architecture-distributed-process-management-concept.png)
+
+Here is an example, using the Marathon service and a user launching a container based on a Docker image:
+
+![Example of distributed process management in DC/OS](../img/dcos-architecture-distributed-process-management-example.png)
+
+The chronological interaction between the above components looks like this. Notice that Executors and Task are folded into one block since in practice this is often the case:
+
+![Sequence diagram for distributed process management in DC/OS](../img/dcos-architecture-distributed-process-management-seq-diagram.png)
+
+In detail, here are the steps:
+
+<table class="table">
+<thead>
+<tr>
+<th>Step</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>1</td>
+<td>Client/Scheduler init: the Client needs to know how to connect to the Scheduler in order to launch a process, for example via Mesos-DNS or DC/OS CLI.</td>
```

Comment:
`...the client needs to know how to connect to the scheduler so that it can launch a process, for example via Mesos-DNS or DC/OS CLI.`